tensorflow/lite/g3doc/inference_with_metadata/task_library/image_classifier.md

```diff
@@ -99,6 +99,140 @@ See the
 [source code and javadoc](https://github.com/tensorflow/tflite-support/blob/master/tensorflow_lite_support/java/src/java/org/tensorflow/lite/task/vision/classifier/ImageClassifier.java)
 for more options to configure `ImageClassifier`.
 
+## Run inference in iOS
+
+### Step 1: Install the dependencies
+
+The task library supports installation using cocoapods. Make sure that cocoapods is installed on your system. 
+Please see the [cocoapods installation guide](https://guides.cocoapods.org/using/getting-started.html#getting-started) for instructions.
+
+Please see the [cocoapods guide](https://guides.cocoapods.org/using/using-cocoapods.html) for details on adding pods to an Xcode project. 
+
+Add the TensorFlowLiteTaskVision pod in Podfile
+
+```
+target 'MyAppWithTaskAPI' do
+  use_frameworks!
+  pod 'TensorFlowLiteTaskVision', '~>0.0.1-nightly.20220508'
+end
+```
+
+Make sure the `.tflite` you will be using for inference is present in your app bundle.
+
+### Step 2: Using the model 
+
+#### Swift
+
+```swift
+// Imports
+import TensorFlowLiteTaskVision
+
+// Initialization
+var imageClassifier: ImageClassifier
+
+guard let modelPath = Bundle.main.path(forResource: "your_model_file_name",
+                                            ofType: "tflite") else {
+    return
+}
+
+let imageClassifierOptions = ImageClassifierOptions(modelPath: modelPath)
+
+// Configure any additional options:
+// imageClassifierOptions.classificationOptions.maxResults = 3
+
+do {
+    imageClassifier = try ImageClassifier.imageClassifier(options: imageClassifierOptions)
+}
+catch {
+    // Handle failure. Check error.localizedDescription to understand the reason for failure.
+}
+
+// Run inference
+// There are other sources for MLImage. For more details, please see:
+// https://developers.google.com/ml-kit/reference/ios/mlimage/api/reference/Classes/GMLImage
+guard let image = UIImage (named: "your_input_image_name"), let mlImage = MLImage(image: image) else {
+    return
+}
+do {
+    let classificationResults: ClassificationResult = try imageClassifier.classify(gmlImage: mlImage)
+}
+catch {
+    // Handle failure.
+}
+```
+
+#### Objective C
+
+```objc
+// Imports
+#import <TensorFlowLiteTaskVision/TFLTaskVision.h>
+
+// Initialization
+NSString *modelPath = [[NSBundle mainBundle] pathForResource:@"your_model_file_name"
+                                                      ofType:@"tflite"];
+
+TFLImageClassifierOptions *imageClassifierOptions =
```

Comment:
Please use shorter name like in the suggestions in the Swift example code to make the code a bit easier to read.
* imageClassifierOptions -> options
* imageClassifier -> classifier